### PR TITLE
[General_layout-navigation] added stack navigators

### PR DESCRIPTION
**Pull request by judygab**
Should we create a bottom tab navigator using react-navigation? It detects which screen is active and can pass customized styling like `activeTintColor` and etc
https://reactnavigation.org/docs/tab-based-navigation#a-stack-navigator-for-each-tab

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,13 +1,50 @@
 import React from 'react';
 import { NavigationContainer, useLinking } from '@react-navigation/native';
-import { createStackNavigator } from '@react-navigation/stack';
+import { createSwitchNavigator, createStackNavigator } from '@react-navigation/stack';
```
Comment (natashkela) [inline App.js:3]:
```suggestion
import {createStackNavigator } from '@react-navigation/stack';
```

File: App.js [hunk 2]
```diff
@@ -1,13 +1,50 @@
...
+      </AppStack.Navigator>
+    );
+  }
+
+  //TODO: pass customized bar components
+  const Tab = createBottomTabNavigator();
+
   return (
       <NavigationContainer>
           <Stack.Navigator initialRouteName="MenuCategoryItems">
```
Comment (natashkela) [inline App.js:50]: ak romel punkcias unda vedzaxdet sinamdvileshi rogor unda davadginot? da magaze info sad shevinaxot, storeshi? 
shegvidzlia rame gavaketot magalitad isAuthenticated?

File: App.js [hunk 3]
```diff
@@ -1,13 +1,50 @@
...
+        <AppStack.Screen name="Delivery" component={Delivery} />
+        <AppStack.Screen name="Account" component={Account} />
+        <AppStack.Screen name="Notification" component={Notification} />
+        <AppStack.Screen name="Cart" component={Cart} />
+      </AppStack.Navigator>
+    );
+  }
+
+  //TODO: pass customized bar components
+  const Tab = createBottomTabNavigator();
```
Comment (natashkela) [inline App.js:46]: es gavaketot dges ertad da ikneb "avamushavot", placeholderebiani screen komponentebit, tavisi headerebit da menu buttonebit. kai progresi ikneba dzaaaaaaaaan amas tu gavaketebt imitoro shen rac momwere mesijebshi magis mixedvit vxvdebi ro dzaan dinamiuri ikneboda 👍

File: App.js (hunk 2 above)
Comment (judygab) [reply App.js:50]: https://reactnavigation.org/docs/auth-flow/
ak naxe gacchnia async storage shi inaxaven tokens an state authenticated air tiara user

Comment (judygab) [reply App.js:50]: https://reactnavigation.org/docs/bottom-tab-navigator/ initiliaze unda gaaketo amisi da mere daudzaxo punkcias wesit

**Pull request merged**